bindings/java/iota-wallet-java/src/main/java/org/iota/Wallet.java

```diff
@@ -0,0 +1,400 @@
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload signTransactionEssence(AccountIdentifier accountIdentifier, SignTransactionEssence method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload submitAndStoreTransaction(AccountIdentifier accountIdentifier, SubmitAndStoreTransaction method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload claimOutputs(AccountIdentifier accountIdentifier, ClaimOutputs method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public void backup(String destination, String password) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("destination", destination);
+        o.addProperty("password", password);
+
+        callBaseApi(new ClientCommand("Backup", o));
+    }
+
+    public void changeStrongholdPassword(String currentPassword, String newPassword) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("currentPassword", currentPassword);
+        o.addProperty("newPassword", newPassword);
+
+        callBaseApi(new ClientCommand("ChangeStrongholdPassword", o));
+    }
+
+    public void clearStrongholdPassword() throws WalletException {
+        callBaseApi(new ClientCommand("ClearStrongholdPassword"));
+    }
+
+    public boolean isStrongholdPasswordAvailable() throws WalletException {
+        return callBaseApi(new ClientCommand("IsStrongholdPasswordAvailable")).getAsBoolean();
+    }
+
+    public void recoverAccounts(int accountGapLimit, int addressGapLimit, SyncOptions sync_options) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("accountGapLimit", accountGapLimit);
+        o.addProperty("addressGapLimit", addressGapLimit);
+        o.add("sync_options", GsonSingleton.getInstance().toJsonTree(sync_options));
+
+        callBaseApi(new ClientCommand("RecoverAccounts", o));
+    }
+
+    public void restoreBackup(String source, String password) throws WalletException {
+        JsonObject o = new JsonObject();
+        o.addProperty("source", source);
+        o.addProperty("password", password);
+
+        callBaseApi(new ClientCommand("RestoreBackup", o));
+    }
+
+    public void removeLatestAccount() throws WalletException {
+        callBaseApi(new ClientCommand("RemoveLatestAccount"));
+    }
+
+    public void deleteAccountsAndDatabase() throws WalletException {
```

Comment:
Will be removed since it's broken https://github.com/iotaledger/wallet.rs/issues/1436

Reply:
fixed